src/main/java/org/junitpioneer/jupiter/json/AbstractJsonArgumentsProvider.java

```diff
@@ -40,7 +42,13 @@ public Stream<Object> provideArguments(ExtensionContext context, Parameter param
 	}
 
 	private Stream<Node> provideNodes(ExtensionContext context) {
-		return provideNodes(context, JsonConverterProvider.getJsonConverter());
+		String objectMapperId = AnnotationSupport
+				.findAnnotation(context.getRequiredTestMethod(), UseObjectMapper.class)
+				.map(UseObjectMapper::value)
+				.orElse("default");
```

Comment:
It could be great if we could change the default one by some [config parameter](https://junit.org/junit5/docs/current/user-guide/#running-tests-config-params). It can be retrieved from the `ExtensionContext` object :)

Reply:
Sounds good.

Reply:
Please check again: Is this what you had in mind? @PrzemyslawSwiderski

Reply:
Yes, works great now :)